ui/admin/app/components/host-catalog-type-badge/index.hbs

```diff
@@ -1,15 +1,4 @@
-{{#if @model.isStatic}}
-  <Rose::Badge>
-    {{t (concat 'resources.host-catalog.types.' @model.compositeType)}}
-  </Rose::Badge>
-{{else if @model.isUnknown}}
-  <Rose::Badge>
-    {{t (concat 'resources.host-catalog.types.' @model.compositeType)}}
-  </Rose::Badge>
-{{else}}
-  <Rose::Badge
-    @icon={{concat 'flight-icons/svg/' @model.compositeType '-color-16'}}
-  >
-    {{t (concat 'resources.host-catalog.types.' @model.compositeType)}}
-  </Rose::Badge>
-{{/if}}
\ No newline at end of file
+<Hds::Badge
+  @icon={{this.icon}}
```

Comment:
Does this work if icon is falsy?

Reply:
Why are Azure and AWS without icons here?  Was this just an example?

Reply:
yes, an example of when I passed in a falsy value.